src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheckTest.java

```diff
@@ -664,10 +658,7 @@ public void testVisitTokenSwitchReflection() {
 
         // Set unsupported option
         final ImportOrderCheck mock = new ImportOrderCheck();
-        final ImportOrderOption importOrderOptionMock = PowerMockito.mock(ImportOrderOption.class);
-        Whitebox.setInternalState(importOrderOptionMock, "name", "NEW_OPTION_FOR_UT");
-        Whitebox.setInternalState(importOrderOptionMock, "ordinal", 5);
-        Whitebox.setInternalState(mock, "option", importOrderOptionMock);
+        TestUtil.getClassDeclaredField(ImportOrderCheck.class, "option").set(mock, null);
```

Comment:
`IllegalStateException` still occurs if we pass option in as `null`.

Reply:
please put a comment above to explain reflection usage, we agreed some time ago that we will do this.

If you additionally remove `@Test(expected = IllegalStateException.class)` ... it would be awesome ...  I thought we already fobid it by some check..... but if you do not want ...  we can postpone this.